kili/authentication.py

```diff
@@ -44,9 +45,9 @@ def __init__(
         if api_endpoint and "v1/graphql" in api_endpoint:
             # pylint: disable=line-too-long
             message = (
-                "We are migrating the API to enhance our service,"
```

Comment:
I think this whole check could be removed (v1/ was deprecated in january 2021 :))